Makefile

```diff
@@ -165,7 +166,8 @@ test/addons/.buildstamp: config.gypi \
 	# Cannot use $(wildcard test/addons/*/) here, it's evaluated before
 	# embedded addons have been generated from the documentation.
 	for dirname in test/addons/*/; do \
```

Comment:
Nit: could we make this `@for` instead to not have the command echoed in the console log?